src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
Same here, you should specify needed properties

Reply:
changed